package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "express-rate-limit",
-	"version": "6.3.0",
+	"version": "6.3.1",
```

Comment:
Please don't bump the version, we will do that as part of the release process later on

Reply:
Sure